internal/driver/glfw/testdata/menu_bar_kbdctrl_traverse_menu_down_2.xml

```diff
@@ -27,12 +27,10 @@
 					<radialGradient centerOffset="0.5,-0.5" pos="-4,29" size="4x4" startColor="shadow"/>
 					<linearGradient angle="270" endColor="shadow" pos="-4,0" size="4x29"/>
 				</widget>
-				<widget size="300x300" type="*glfw.menuBarUnderlay">
+				<widget size="300x300" type="*glfw.menuBarBackground">
```

Comment:
this type no longer exists

Reply:
This is very strange. I will try syncing again